generate-turso-db.py

```
@@ -8,6 +8,9 @@
 import json
 from datetime import datetime
 from pathlib import Path
+import math
+import threading
```

Ruff (0.11.9)

`threading` imported but unused

Remove unused import: `threading`

(F401)